app/models/supplier.rb

```diff
@@ -6,19 +6,22 @@ class Supplier < ActiveRecord::Base
   # save lists in an array in database
   serialize :lists
 
+  FTP_TYPES = ['bnn', 'foodsoft'].freeze
```

Comment:
Why not all types? Maybe I'm missing something :)

Reply:
My foodcoops needs the feature for foodsoft files. I have tested this change for BNN files and for foodsoft files.

After this PR, we can easily add more formats. I suggest that we enable a new format when we have a use case (and hence some test data).

Reply:
Ok. I'd be ok for enabling all file formats, since all existing BNN syncs will keep working, and setting up a new one needs testing anyway, and one would see it if it doesn't work.
But fine for now!